### ISSUE TITLE
when the object missed in the screen, the siamese-rpn still track other object

### ISSUE THREAD

**Issue by destinykela** [open-mmlab/mmtracking#673]
I used the siamese_rpn_r50_20e_lasot_20220420_181845-dd0f151e.pth to inference the demo_sot.py. But when object misses in the 
screen, the tracker will track other object which is similar to the origin.

**Comment by JingweiZhang12** [comment]
Indeed. This will not affect the evaluation performance on SOT datasets since the frames where object miss will not be evaluated.
If you want to track in the reality scene, you can use the classification score to verify the confidence of object existence.

**Comment by destinykela** [comment]
@JingweiZhang12  Thank you, doctor zhang, but i can't find the place to modify the classification score. Could you tell me how to modify the classification score or should I  do post-processing by myself?

**Comment by JingweiZhang12** [comment]
Here, https://github.com/open-mmlab/mmtracking/blob/master/mmtrack/models/sot/siamrpn.py#L244
